#!/usr/bin/env python
PACKAGE = "parsian_protobuf_wrapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("vision_multicast_port", int_t,    0, "SSL-Vision Multicast Port", 10006,  0, 66000)
gen.add("vision_multicast_ip"  , str_t,    0, "SSL-Vision Multicast IP",  "224.5.23.2")

exit(gen.generate(PACKAGE, "parsian_protobuf_wrapper", "SSLVision"))
